docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGBMRanking.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
```

Comment:
>Microsoft.ML.Samples.Dynamic [](start = 10, length = 28)

please change the namespace to Microsoft.ML.Samples.Dynamic.Trainers.Ranking and rename the class from LightGbmRanking to just LightGbm. Please do the same for the BinaryClassification examples.